src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -93,12 +95,12 @@ public void register(Router router) {
     router.put(RELATIVE_ITEMS_PATH_ID).handler(this::update);
     router.delete(RELATIVE_ITEMS_PATH_ID).handler(this::deleteById);
 
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-in-process")
-      .handler(handle(this::markAsInProcess));
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-in-process-non-requestable")
-      .handler(handle(this::markAsInProcessNonRequestable));
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-intellectual-item")
-      .handler(handle(this::markAsIntellectualItem));
+    Arrays.stream(ItemStatusName.values()).map(itemStatusName -> ItemStatusURL.getUrlForItemStatusName(itemStatusName))
+      .filter(itemStatusUrl -> itemStatusUrl.isPresent())
+      .forEach(itemStatusUrl -> registerMarkItemAsHandler(itemStatusUrl.get(),router));
```

Comment:
FOLIO's code standards state that their should be a space after the comma separating parameters 

```suggestion
      .forEach(itemStatusUrl -> registerMarkItemAsHandler(itemStatusUrl.get(), router));
```